pkg/cgroups/cgroups.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	CPUPeriod = 1000000 // one million microseconds
+	CPUPeriod = 100000 // one hundred thousand microseconds
```

Comment:
Does this change the cpu quote of applications?
i.e. in `pkg/cgroups/cgroups.go` line 52: `quote := cpus * CPUPeriod`

Reply:
it shouldn't -- quota is a fraction of period, so changing the one constant makes both scale appropriately